api/src/test/kotlin/edu/wgu/osmt/richskill/RichSkillEsRepoTest.kt

```diff
@@ -375,11 +375,12 @@ class RichSkillEsRepoTest @Autowired constructor(
     @Test
     fun testByApiSearchWithAdvance() {
         // Arrange
+        val authorsList = listOf("author")
```

Comment:
Good consideration. I'm not sure that this one should be in scope for this issue. I'd love to identify whether author is the only scenario where this type of keyword-based searching would cause false positives due to search finding responsive items where terms appeared across multiple distinct values.